progfun/funcional_teoria/index.html

```diff
@@ -588,10 +604,16 @@ <h3 class="text-center">Paralelismo</h3>
                         <p>
                             Dado que en la programacion funcional no existe un estado global no es necesario implementar
                             las tecnicas de paralelismo como
-                            (Multexes , Semaforos ,Locks) por lo cual se pueden ralizar funciones en paralelo siempre q
+                            (Multexes , Semaforos ,Locks) por lo cual se pueden ralizar funciones en paralelo siempre que
```

Comment:
```suggestion
                            (Multexes, Semáforos, Locks) por lo cual se pueden realizar funciones en paralelo siempre que
```
Por favor también revisar la ortografía en este párrafo.